subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1005,12 +1009,29 @@ static void le_create_connection(struct net_buf *buf, struct net_buf **evt)
 static void le_create_conn_cancel(struct net_buf *buf, struct net_buf **evt)
 {
 	struct bt_hci_evt_cc_status *ccst;
+	struct net_buf *cc;
+	u8_t cmd_status;
 	u32_t status;
 
 	status = ll_connect_disable();
+	cmd_status = (!status) ? 0x00 : BT_HCI_ERR_CMD_DISALLOWED;
```

Comment:
I realize this is copy-paste of existing code, but I wonder why it's using negated logic. I'd instead do it as `cmd_status = status ? : BT_HCI_ERR_CMD_DISALLOWED : 0x00;`. IIRC the idea was to make the ll APIs return the u8_t HCI status directly in the future, right?

Reply:
Yes, that is correct. Fixed, thanks!